openapi-codegen/sbt-plugin/src/sbt-test/sbt-openapi-codegen/oneOf-json-roundtrip/swagger.yaml

```diff
@@ -63,22 +63,13 @@ paths:
     put:
       responses:
         '200':
-          description: a non-optional response body
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/NotNullableThingy'
+          $ref: '#/components/responses/optionalityTest'
       requestBody:
-        required: false
-        description: an optional request body (not required)
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/NotNullableThingy'
```

Comment:
Generated code is completely unchanged by this, which is exactly the behaviour I was aiming for